root_orchestrator/resource-abstractor/app.py

```diff
@@ -0,0 +1,41 @@
+import os
+from flask import Flask
+from flask_smorest import Api
+from flask_swagger_ui import get_swaggerui_blueprint
+
+from db.mongodb_client import mongo_init
+from api.v1 import blueprints
+
+MY_PORT = os.environ.get("MY_PORT")
+
+app = Flask(__name__)
+
+app.config['OPENAPI_VERSION'] = '3.0.2'
+app.config['API_TITLE'] = 'Resource Abstractor Api'
+app.config['API_VERSION'] = 'v1'
+app.config["OPENAPI_URL_PREFIX"] = '/docs'
+
+api = Api(app)
+mongo_init(app)
+
+# Register blueprints
+SWAGGER_URL = '/api/docs'
+API_URL = '/docs/openapi.json'
+swaggerui_blueprint = get_swaggerui_blueprint(
+    SWAGGER_URL,
+    API_URL,
+    config={
+        'app_name': "Resource Abstractor"
+    },
+)
+app.register_blueprint(swaggerui_blueprint)
+
+for blp in blueprints:
+    api.register_blueprint(blp)
+
+@app.route('/', methods=['GET'])
+def hello_world():
+    return "Hello, World! This is the Resource Abstractor.\n"
```

Comment:
I assume this is WIP? Maybe add a TODO here or remove it?

Reply:
I would keep an empty debug endpoint like '/'. If we insert a health check, this can be used to test the response 200 from the service. What do you think?

Reply:
That makes sense, I would prefer to rename/refactor it then so it is obvious that it is a health check and not a potentially forgotten hello_world. 😅

Reply:
Agreed, `health` check point makes more sense.